src/ESLint.Formatter/package.json

```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "chai": "^4.2.0",
     "jschardet": "latest",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.14",
```

Comment:
>^4.17.1 [](start = 15, length = 7)

Do we have a release notes document for this component that we can/should update?

Reply:
We don't. @rscrivens: As a separate PR, would you please:

1. Place a ReleaseHistory.md in the src\ESLint.Formatter\ directory, modeled on the one in the src\ directory? (Just start with this release, fixing the vuln, but we won't publish it yet.)
2. Add it to package.json.

The next time we publish (the time _after_ this vuln-fixing release), it will include the release history.